### PR TITLE
Add integrator multi dim tutorial

### PR DESCRIPTION
**Pull request by atolosadelgado**
Fixes https://github.com/root-project/root/issues/7740

# This Pull request:

## Changes or fixes:
Add tutorial about multi dimensional integrator

## Checklist:

- [x ] tested changes locally
- [ ] updated the docs (if necessary)

This PR fixes # 
https://github.com/root-project/root/issues/7740

### CODE REVIEW

File: tutorials/math/mathmoreIntegrationMultidim.C [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+/// Example on the usage of the multidimensional integration algorithm of MathMore
+/// Please refer to the web documentation for further details: 
+///      https://root.cern.ch/root/htmldoc/guides/users-guide/MathLibraries.html#numerical-integration
+/// To execute the macro type the following:
+///
+/// ~~~{.cpp}
+/// root[0] .x mathmoreIntegrationMultidim.C
+/// ~~~
+///
+/// This tutorials require having libMathMore built with ROOT.
```
Comment (ferdymercury) [inline tutorials/math/mathmoreIntegrationMultidim.C:13]:
```suggestion
/// This tutorial requires having libMathMore built with ROOT.
```

**Comment by hahnjo** [comment]
@phsft-bot build

**Comment by dpiparo** [comment]
@atolosadelgado : we are 1 typo away from merging the PR and closing the issue. Do you have a moment to address the minimal comment which was made?

**Comment by atolosadelgado** [comment]
Hi,

I fixed the typo in the new tutorial, and in the older tutorial from which I copypasted the header :)

**Comment by dpiparo** [comment]
triggered building and testing. If everything goes well, for me the PR is ready to be merged!

**Review by ferdymercury — APPROVED** [review]

**Comment by vepadulano** [comment]
@phsft-bot build

**Comment by vepadulano** [comment]
@phsft-bot build

**Comment by hahnjo** [comment]
@vepadulano this PR doesn't have Philippe's fix and therefore blocks the `mac12` and `mac13` nodes (until I notice and manually cancel the jobs). If this needs more CI runs, please consider rebasing on latest `master` to avoid that...

**Comment by atolosadelgado** [comment]
> @vepadulano this PR doesn't have Philippe's fix and therefore blocks the `mac12` and `mac13` nodes (until I notice and manually cancel the jobs). If this needs more CI runs, please consider rebasing on latest `master` to avoid that...

Thank you for noticing. I have just sync the branch, I hope now the tests pass.

**Pull request merged**